haskell_scotty/Client.hs

```diff
@@ -59,3 +64,13 @@ instance FromJSON Client where
       (attributes >>= (.: "phone")) <*>
       (attributes >>= (.: "created_at")) <*>
       (attributes >>= (.: "updated_at"))
+  parseJSON _ = error "unexpected non-object JSON"
+
+instance QueryResults Client where
+  convertResults fs vs =
+      Client $... constructorWrap (undefined :: Client) fs vs
+
+instance QueryParams Client where
+  renderParams
+    (Client _ name' address' city' state' zip' country' email' contact' phone' created_at' updated_at') =
```

Comment:
Same suffix notation again, but it doesn't appear to be accounting for mutation. Is this just to avoid name collision with the methods defined on `Client`?